ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualLedgerTransaction.kt

```diff
@@ -34,10 +34,10 @@ interface ConsensualLedgerTransaction {
     val id: SecureHash
 
     /**
-     * @property requiredSignatories Set of [PublicKey] needed to make the underlying transaction valid.
+     * @property requiredSignatories List of [PublicKey] needed to make the underlying transaction valid.
      * Essentially the union of the participants of the transaction's [ConsensualState]s.
      */
-    val requiredSignatories: Set<PublicKey>
+    val requiredSignatories: List<PublicKey>
```

Comment:
Why are we changing this? We need lists on stuff stored directly in signed/wire transaction to ensure consistent signing, but here this is a synthesized value and easiest to manipulate as a Set

Reply:
I have reverted this.